sunpy/data/test/aia_171_level1_roll.fits

```diff
@@ -0,0 +1,9 @@
                               AGT4SVZ =                  -42                                                  AIMGSHEN=                   13                                                  KEYWDDOC= 'http://www.lmsal.com/sdodocs/aiafitskeywords.pdf'                    RECNUM  =            135407475                                                  BLANK   =               -32768                                                  DETECTOR= 'AIA     '                                                            COMMENT FITS (Flexible Image Transport System) format is defined in 'Astronomy  COMMENT and Astrophysics', volume 376, page 359; bibcode: 2001A&A...376..359H   HISTORY                                                                         END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             ��      ?�      ��              ��      ?�      ?�              ?�      ?�      ?�      @
```

Comment:
What is the purpose of including a whole new test file, instead of just rotating an existing test map?

Reply:
Donno, I had the roll file so I used it. It will be a little faster on setup of all the tests that use it I guess :rofl:

Reply:
Given we try to avoid clogging the repo with image files, I think I am pro removing this and rotating a pre-existing map in a test fixture.